user_guide_src/source/libraries/pagination.rst

```diff
@@ -291,7 +291,7 @@ page is page 3. If the surrounding count is 2, then the following links would sh
 ``hasPreviousPage()`` would return ``true`` since there is page 2. And,
 ``hasNextPage()`` would return ``true`` since there is page 4.
 
-.. note:: The difference to `hasPrevious() & hasNext()`_ is that they are based
+.. note:: The difference to `hasPreviousPage() & hasNextPage()`_ is that they are based
```

Comment:
While the original text is something wrong, but is this really correct?
This is in the section of `hasPreviousPage() & hasNextPage()`.

Reply:
Reverted the change.